index.js

```diff
@@ -0,0 +1,68 @@
+let addMessage = document.querySelector(".myInput"),
+  addButton = document.querySelector(".add"),
+  todo = document.querySelector(".todo"),
+  deleteButton = document.querySelector(".delete");
+
+let todoList = [];
+
+addButton.addEventListener("click", () => {
+  let newTodo = {
```

Comment:
If addMessage.value is empty then it doesnt make sense to create object. Move checking below to top of function